.github/workflows/ci.yml

```diff
@@ -33,3 +33,23 @@ jobs:
 
             - name: Run Phing
               run: composer run-script phing
+
+    phpstan:
+        runs-on: ubuntu-latest
+        name: Static Analysis
+        steps:
+            - name: 📤 Checkout project
+              uses: actions/checkout@v2
+
+            - name: 🐘 Install PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '7.4'
+                  tools: composer:v2
+
+            - name: 📦 Install dependencies
+              if: steps.composer-cache.outputs.cache-hit != 'true'
```

Comment:
I think this `if` can be removed, also from the installation of dependencies in the other job. I removed the caching in a previous PR because this library has no `.lock` file so the cache never got invalidated. And we can't cache based on `composer.json`, because then it wouldn't install the latest versions (matching the constraints of the versions in composer.json) because it would have cached older versions and not invalidate them because `composer.json` didn't change, but there _are_ newer versions available which would be installed in other circumstances like when someone adds this package to a project.

Long story short: I don't think we should cache composer dependencies if we don't have a `.lock` file.

And the reason we don't have a `.lock` file in our libraries is so we don't get a false sense of security that everything works, because when you add this package to a project the `.lock` file in it will be ignored and a newer version of a dependency might be installed.